docs/two-phase-commit-transactions.md

```diff
@@ -183,9 +183,49 @@ Similar to `prepare()`, you can call `validate()` in the coordinator/participant
 Currently, you need to call `validate()` when you use the `Consensus Commit` transaction manager with `EXTRA_READ` serializable strategy in `SERIALIZABLE` isolation level.
 In other cases, `validate()` does nothing.
 
-### Suspend and resume the transaction
+### Load balancing in Two-phase Commit Transactions
+
+Usually, services using Two-phase Commit Transactions achieve a transaction while exchanging multiple requests/responses as follows:
+
+<p align="center">
+<img src="images/two_phase_commit_sequence_diagram.png" width="400" />
+</p>
+
+Also, a service typically has multiple hosts/servers for scalability and availability, and uses server-side (proxy) or client-side load balancing to distribute the requests to the hosts/servers.
+Since a transaction processing in Two-phase Commit Transactions is stateful, operations in a transaction must be executed in the same hosts/servers.
+In other words, the load balancing must distribute the requests in a transaction to the same host/server.
+
+<p align="center">
+<img src="images/two_phase_commit_load_balancing.png" width="500" />
```

Comment:
In this schema, there is a missing `u` in the `reqest` word placed on the arrow from Client to Service A.

Reply:
Thank you for pointing it out! I fixed it in 08ba4508825c01587e16fed3b8fc856f368c9ded.